test/e2e/collect-ci-artifacts.sh

```diff
@@ -0,0 +1,52 @@
+#! /bin/bash
+
+set -o pipefail
+set -o nounset
+set -o errexit
+
+: "${KUBECONFIG:?}"
```

Comment:
Do we need/use the `KUBECONFIG` variable?

Reply:
Nit: comment to that effect then? Otherwise someone might decide to delete it in the future.